openapi.yaml

```diff
@@ -313,6 +313,16 @@ paths:
         schema:
           type: integer
           format: int64
+    patch:
```

Comment:
Patch may not be the right choice? There's also a potential for race conditions with this, so locking may be needed in the implementation.

Reply:
The race condition is a broader problem than this change. I will create a new issue.